.gitignore

```diff
@@ -23,3 +23,7 @@
 /.node_modules.ember-try/
 /bower.json.ember-try
 /package.json.ember-try
+
+# lockfiles -- this is a library; lock files are more apt to mislead than help
```

Comment:
<3
This is the same approach ruby/gems have!

Reply:
Hmm, ya don’t think we should do this. All of our ember-try scenarios already test with floating deps / no lock file, and our linting / basic tests should always be running against known good deps.

Reply:
@rwjblue, I'm happy to invert it; this was just originally the recommendation I saw for libraries from Yarn itself. Will push a version with the lockfile!

Reply:
> this was just originally the recommendation I saw for libraries from Yarn itself

Ya, I also recall some confusion around that at initial release time, but I believe it is pretty well cleared up at this point. Let’s keep this close to the ember-cli addon defaults (not sure if the stock one is good enough for us, but we can make fewer tweaks).

Some references:

https://classic.yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control


https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
Resolved as (force pushed) 8e3bcd2!